test/ConfigResourceFactoryTest.php

```diff
@@ -10,19 +10,19 @@
 
 use Interop\Container\ContainerInterface;
 use Laminas\ApiTools\Configuration\ConfigResource;
-use Laminas\ApiTools\Configuration\ConfigWriter;
 use Laminas\ApiTools\Configuration\Factory\ConfigResourceFactory;
 use Laminas\Config\Writer\WriterInterface;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
-use Prophecy\Prophecy\ProphecyInterface;
-use Prophecy\PhpUnit\ProphecyTrait;
 
 class ConfigResourceFactoryTest extends TestCase
 {
-    use ProphecyTrait;
+    /** @var string */
+    private const WRITER_SERVICE = 'Laminas\ApiTools\Configuration\ConfigWriter';
```

Comment:
Could this use the `::class` syntax?

Reply:
Psalm was complaining about it, as the class doesn't actually exist; it's a pseudo-class for specifying a config writer for the container.